src/helpers/api/axios.js

```diff
@@ -0,0 +1,7 @@
+const instance = {
+  headers: {
+    'Content-Type': 'application/json',
+  },
+};
+
+export default instance;
```

Comment:
Would you be kind to explain what this whole file does since you are not importing it anywhere else?